lib/ansible/modules/monitoring/grafana_dashboard.py

```diff
@@ -288,6 +364,7 @@ def main():
             grafana_password=dict(default='admin', no_log=True),
             grafana_api_key=dict(type='str', no_log=True),
             org_id=dict(default=1, type='int'),
+            uid=dict(type='str'),
```

Comment:
shouldn't did and slug be added in the mutually_exclusive list ?

Reply:
Yep, good point !